lib/gtl_formatter.rb

```diff
@@ -9,7 +10,18 @@ class GtlFormatter
   COLUMN_WITH_ICON = {
     'authentication_status'  => 'authentication_status_image',
     'last_compliance_status' => 'last_compliance_status_image',
-    'normalized_state'       => 'normalized_state_image'
+  }.freeze
+
+  COLUMN_WITH_BACKGROUND_ICON = {
+    'normalized_state' => 'normalized_state_image'
+  }.freeze
+
+  COLUMN_WITH_OS_ICON = {
+    'os_image_name' => 'os_info_details'
+  }.freeze
+
+  COLUMN_WITH_OS_TEXT = {
+    'product_name' => 'product_name_image'
```

Comment:
If I'm understanding, these are all hashes of key beings the column name and value being the method to call, right?

I wonder if we can make that more obvious or better yet, create aliases or something so there's no need to translate a column to a method.

Reply:
You are right, we don't need this function at all as it just printing out the value, removing it.